test/e2e/discovery/service_discovery.go

```diff
@@ -53,6 +53,10 @@ func RunServiceDiscoveryTest(f *lhframework.Framework) {
 	By(fmt.Sprintf("Creating a Netshoot Deployment on %q", clusterAName))
 	netshootPodList := f.NewNetShootDeployment(framework.ClusterA)
 
+	if svc, err := f.GetService(framework.ClusterB, nginxServiceClusterB.Name, nginxServiceClusterB.Namespace); err == nil {
```

Comment:
Don't you need to wait until service has the global IP for global IP ?

Reply:
> Don't you need to wait until service has the global IP for global IP ?

Yes. For that I need globalnetEnabled flag to be set in context. I discussed with Sridhar and will add that to shipyard code. I think it is common enough that we shouldn't duplicate in submariner and lighthouse both.

Reply:
https://github.com/submariner-io/shipyard/issues/72 - it's on my plate for 0.5.0.  I can expedite it.